python/setup.py

```diff
@@ -23,12 +23,6 @@
         'dataclasses-json==0.3.0',
         'grpclib==0.2.4',
         'protobuf==3.7.1',
-    ],
-    setup_requires=[
```

Comment:
setup_requires is for extra packages that are not required for bundling and installing the library: https://blog.godatadriven.com/setup-py

no need to have all packages duplicated here